tx_json.go

```diff
@@ -0,0 +1,457 @@
+package wavelet
+
+import (
```

Comment:
Would recommend `quick.Check` tests to ensure that the JSON payloads for each individual tag/payload pair does not cause any panics. https://golang.org/pkg/testing/quick/